Fail to build manylinux1 wheels

There is an issue building `manylinux1` wheels with current master.
Compilation of `cblosc` fails:
```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DHAVE_LZ4=1 -DHAVE_ZLIB=1 -DHAVE_ZSTD=1 -DH5_USE_18_API -DH5_USE_18_API -DSHUFFLE_SSE2_ENABLED=1 -Isrc/hdf5/include -Isrc/hdf5/include/linux -Isrc/c-blosc/ -Isrc/c-blosc/blosc -Isrc/c-blosc/internal-complibs/lz4-1.9.2 -Isrc/c-blosc/internal-complibs/zlib-1.2.8 -Isrc/c-blosc/internal-complibs/zstd-1.4.5 -Isrc/c-blosc/internal-complibs/zstd-1.4.5/common -Isrc/hdf5-blosc/src/ -I/opt/python/cp37-cp37m/include/python3.7m -c src/c-blosc/blosc/blosclz.c -o build/temp.linux-x86_64-3.7/src/c-blosc/blosc/blosclz.o -msse2
src/c-blosc/blosc/blosclz.c: In function ‘get_csize’:
src/c-blosc/blosc/blosclz.c:409:3: error: ‘for’ loop initial declarations are only allowed in C99 mode
   for (unsigned i = 0; i < (1U << HASH_LOG); i++) {
   ^
src/c-blosc/blosc/blosclz.c:409:3: note: use option -std=c99 or -std=gnu99 to compile your code
src/c-blosc/blosc/blosclz.c: In function ‘blosclz_compress’:
src/c-blosc/blosc/blosclz.c:534:3: error: ‘for’ loop initial declarations are only allowed in C99 mode
   for (unsigned i = 0; i < (1U << hashlog); i++) {
   ^
src/c-blosc/blosc/blosclz.c:643:47: warning: signed and unsigned type in conditional expression [-Wsign-compare]
     unsigned minlen = (clevel == 9) ? ipshift : minlen_[clevel];
                                               ^
error: command 'gcc' failed with exit status 1
```

Full log: https://gitlab.esrf.fr/silx/bob/hdf5plugin/-/jobs/174560

--- a/setup.py
+++ b/setup.py
@@ -1,7 +1,7 @@
 # coding: utf-8
 # /*##########################################################################
 #
-# Copyright (c) 2016-2020 European Synchrotron Radiation Facility
+# Copyright (c) 2016-2021 European Synchrotron Radiation Facility
 #
 # Permission is hereby granted, free of charge, to any person obtaining a copy
 # of this software and associated documentation files (the "Software"), to deal
@@ -436,6 +436,7 @@ include_dirs += glob(blosc_dir + 'internal-complibs/zstd*')
 include_dirs += glob(blosc_dir + 'internal-complibs/zstd*/common')
 define_macros.append(('HAVE_ZSTD', 1))
 
+extra_compile_args = ['-std=c99']  # Needed to build manylinux1 wheels
 
 blosc_plugin = HDF5PluginExtension(
     "hdf5plugin.plugins.libh5blosc",
@@ -445,6 +446,7 @@ blosc_plugin = HDF5PluginExtension(
         prefix(hdf5_blosc_dir, ['blosc_filter.h', 'blosc_plugin.h']),
     include_dirs=include_dirs + [hdf5_blosc_dir],
     define_macros=define_macros,
+    extra_compile_args=extra_compile_args,
     sse2=sse2_kwargs,
     avx2=avx2_kwargs,
     cpp11=cpp11_kwargs,

--- a/setup.py
+++ b/setup.py
@@ -436,7 +436,7 @@ include_dirs += glob(blosc_dir + 'internal-complibs/zstd*')
 include_dirs += glob(blosc_dir + 'internal-complibs/zstd*/common')
 define_macros.append(('HAVE_ZSTD', 1))
 
-extra_compile_args = ['-std=c99']  # Needed to build manylinux1 wheels
+extra_compile_args = ['-std=gnu99']  # Needed to build manylinux1 wheels
 
 blosc_plugin = HDF5PluginExtension(
     "hdf5plugin.plugins.libh5blosc",

